cellpack/autopack/Environment.py

```diff
@@ -1605,9 +1606,8 @@ def buildGrid(
             # save bb for current fill
             self.log.info("####BUILD GRID - step %r", self.smallestProteinSize)
             self.fillBB = boundingBox
-            self.grid = Grid(
-                boundingBox=boundingBox, space=self.smallestProteinSize, lookup=lookup
-            )
+            spacing = self.smallestProteinSize * 1.1547
```

Comment:
Just worked through the significance of 1.1547 = (2/sqrt(3)). This ensures that the smallest spherical protein completely encircles a cube made out of grid points with the given spacing, while touching all 8 corners. Mentioning it here for future reference.

Reply:
yeah, so it was how the grid was always supposed to be working, but if you look through the code it got removed and added all over the place... partially I think because of "smallestProteinSize" and "space" being used interchangeably. So I just do it once here when the "spacing" is being sent in